tests/ignite/metrics/nlp/test_bleu.py

```diff
@@ -28,6 +29,32 @@ def test_wrong_inputs():
     with pytest.raises(NotComputableError):
         Bleu().compute()
 
+    with pytest.raises(ValueError, match='Average must be either "macro" or "micro"'):
+        Bleu(average="macros")
+
+
+@pytest.mark.parametrize(
```

Comment:
I was thinking we could factorize a bit the code 
```python
parametrize_arguments = (
    "candidate, references",
    [
        ([["a"], ["a"]]),
        ([["a", "a", "a", "b", "c"]], [[["a", "b", "c"], ["a", "a", "d"]]]),
        corpus.sample_1,
        corpus.sample_2,
        corpus.sample_3,
        corpus.sample_4,
    ],
)

@pytest.mark.parametrize(*parametrize_arguments)
def test_sentence_bleu(candidate, references):
    ....

@pytest.mark.parametrize(*parametrize_arguments)
def test_corpus_bleu(candidate, references):
    ....
```

Reply:
This is clever :) 
I have incorporated it.